practice-typescript/src/scripts/constants/user.ts

```diff
@@ -0,0 +1,109 @@
+import User from 'scripts/types/user';
+
+// Constant specifying the maximum length allowed for input fields given first and last name
+const MAX_LENGTH_NAME = 12;
```

Comment:
Naming for this is not good enough. Let's think about it and fix it later. About the value of max length: 12 => it's too short. You can research around to know how it should be

Reply:
fixed